dot.go

```diff
@@ -30,8 +31,17 @@ func dotToImage(dot []byte) (string, error) {
 		dotExe = dot
 	}
 
-	img := filepath.Join(os.TempDir(), fmt.Sprintf("go-callvis_export.svg"))
-	cmd := exec.Command(dotExe, "-Tsvg", "-o", img)
+	if format != "svg" && format != "png" {
```

Comment:
The dot program actually supports a lot of output formats, not just `svg` and `png`. See all supported output formats here: https://www.graphviz.org/doc/info/output.html

Reply:
I'll read up on it and try to make the output format support more open-ended then.